src/tests/ftest/process_core_files.py

```diff
@@ -382,8 +382,6 @@ def resolve_debuginfo(self, pkg):
             command = " ".join(
                 ["rpm", "-q", "--qf", "'%{name} %{version} %{release} %{epoch}'", pkg])
             result = run_local(self.log, command)
-            if not result.passed:
-                raise RunException(f"Error running {command}")
```

Comment:
If this fails, `name, version, release, epoch = result.joined_stdout.split()` would probably be wrong and this function would still return `package_info` with weird data

Reply:
Thanks, code updated.